Mapping/Driver/AttributeDriver.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the BazingaGeocoderBundle package.
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @license    MIT License
+ */
+
+namespace Bazinga\GeocoderBundle\Mapping\Driver;
+
+use Bazinga\GeocoderBundle\Mapping\Annotations;
+use Bazinga\GeocoderBundle\Mapping\ClassMetadata;
+use Bazinga\GeocoderBundle\Mapping\Exception\MappingException;
+use Doctrine\Persistence\Proxy;
+
+/**
+ * @author Pierre du Plessis <pdples@gmail.com>
+ */
+class AttributeDriver implements DriverInterface
```

Comment:
Do we want this driver to be extendable?

Reply:
I've made this class final now